Feature/adding log export tutorial (#223)
ausaccessfed/dev-portal: docs/_exporting-identity-provider-logs/01-overview.md

---
title: Overview
order: 1
duration: 1
last_updated: 15 April, 2026
---

This tutorial shows how to export an identity providers logs and import them into certain services.

## What you'll learn

- How to export identity provider logs.
- How to import logs from the api into azure sentinel.

## What you'll need

- Completion of the [Log into Federation Manager Tutorial](/log-into-federation-manager/01-overview)
- Completion of the [Federation Manager API Tutorial](/log-into-federation-manager/04-using-the-api)

> - Request API Access and log export features, at the moment both are feature flagged please send an email to <mailto:support@aaf.edu.au> for access.

Reviewer: ```suggestion
- Request API Access and log export features, by sending an email to <mailto:support@aaf.edu.au>
```

Reviewer: Could also add here that you need to be the admin of your IdP, if your IdP is missing fromt he UI, please reach out to support?

Author: idk i guess id assume this would be a preface, like why are you requesting logs for an idp you shouldnt have access to?
